README.md

```diff
@@ -45,3 +66,19 @@ Therefore, block witnesses would allow stateless nodes to store only state roots
 * https://medium.com/@mandrigin/semi-stateless-initial-sync-experiment-897cc9c330cb
 * https://ethresear.ch/t/some-quick-numbers-on-code-merkelization/7260
 * https://medium.com/ewasm/evm-bytecode-merklization-2a8366ab0c90
+
+### Sync and Network
```

Comment:
Maybe this whole "Further Reading" section is redundant with the roadmap doc. Maybe it should all go in the roadmap doc.

Reply:
I'm reluctant to turn the roadmap doc into a bucket for everything; I think it's helpful to have a further reading section that allows someone just trying to skim several relevant pages to get a sense of the scope of the problem. I suppose a more verbose source list could be in the techTree or roadmap doc, too

Reply:
Oops, I meant that the "Further Reading" section may be redundant with the proposed tech tree doc. I did not consider the roadmap doc or the proposed glossary https://github.com/ethereum/stateless-ethereum-specs/issues/44. If there is lots of redundancy, then we should consider merging some of these docs. We can iterate on this.